llvm/lib/Transforms/Vectorize/LoopVectorizationPlanner.h

```diff
@@ -25,7 +25,7 @@
 #define LLVM_TRANSFORMS_VECTORIZE_LOOPVECTORIZATIONPLANNER_H
 
 #include "VPlan.h"
-#include "llvm/ADT/SmallSet.h"
```

Comment:
Is this intentional?

Reply:
Just thought I'd clean up unused includes when making a change to the file: can revert if undesired?